tests/components/test_rflink.py

```diff
@@ -285,14 +273,14 @@ def test_error_when_not_connected(hass, monkeypatch):
     failures = [False, True, False]
 
     # setup mocking rflink module
-    _, _, _, disconnect_callback = yield from mock_rflink(
+    _, _, _, disconnect_callback = await mock_rflink(
         hass, config, domain, monkeypatch, failures=failures)
 
     # rflink initiated disconnect
     disconnect_callback(None)
 
-    yield from asyncio.sleep(0, loop=hass.loop)
+    await asyncio.sleep(0, loop=hass.loop)
```

Comment:
Try replace this with `await hass.async_block_till_done`. It will sleep at least once.

Reply:
Why do we need the sleep?

Reply:
It works with your suggestion (`await hass.async_block_till_done()`). 
But it also works by completely eliminating the sentence.

Does it seem appropriate if we eliminate it? (I'm sorry I do not help more on this)

Reply:
> (I'm sorry I do not help more on this)

Ouch! That sounds very rude.
What I meant to say is that I **can not** help at this topic